base/reflection.jl

```diff
@@ -672,7 +672,7 @@ function signature_type(@nospecialize(f), @nospecialize(args))
 end
 
 """
-    code_lowered(f, types, expand_generated = true)
+    code_lowered(f, types; expand_generated = true)
```

Comment:
Maybe this one could just be `generated`? That would parallel `if @generated`.

Reply:
I commented about this here: https://github.com/JuliaLang/julia/issues/25188#issuecomment-359352314. I'm not too passionate though so if people feel like `generated` makes sense I can change it.